kt-extensions/src/main/java/com/mindera/skeletoid/kt/extensions/utils/Parcel.kt

```diff
@@ -16,10 +16,6 @@ inline fun <reified T> parcelableCreator(
             override fun newArray(size: Int) = arrayOfNulls<T>(size)
         }
 
-fun Parcel.readBoolean() = readInt() != 0
-
-fun Parcel.writeBoolean(value: Boolean) = writeInt(if (value) 1 else 0)
```

Comment:
not needed anymore?

Reply:
Yep, the Parcel now has this exact implementation.